docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Gift Cards
+---
+
+:::note
+Gift cards are available in Saleor 3.1.
+:::
+
+This page contains information on how to manage gift cards by selecting a specific currency, sending them to customers, or setting expiring dates.
+
+## Introduction
+Gift cards are timeless and universal gifts. They are suitable for customers, employees, CEOs, business partners, or family members. Issuing gift cards can increase your sales and provide a better customer experience.
+
+## How to
+### How to create gift cards?
+Once you have created a product type that can be used as a gift card or voucher, you can issue a gift card. Go to the Gift Card page of the Catalog section. 
```

Comment:
I don't think you need a product type of gift card or voucher to create a gift card. There are product types of gift card but it's for selling gift cards to customers in the store. You can create a gift card not as a product and just give it to someone eg. as a compensation.

Reply:
cc: @bmigirl

Reply:
Good point. I'll change it.

Reply:
Yes, you need a gift card product type to sell a gift card as a product, but no need of it to just issue it manually from gift cards list etc.

Reply:
Also there is a mention of `Once you have created a product type` - I think this should be a link to an appropriate section of product type docs